js/models/message_handler.js

```diff
@@ -29,14 +29,56 @@ export default class MessageHandler {
     this.aladin.addMOC(A.MOCFromJSON(msg["moc_dict"], options));
   }
 
-  handleAddOverlayFromSTCS(msg) {
-    const overlayOptions = convertOptionNamesToCamelCase(
-      msg["overlay_options"] || {},
+  handleAddOverlay(msg) {
+    const regions = msg["regions_infos"];
+    const graphic_options = convertOptionNamesToCamelCase(
+      msg["graphic_options"] || {},
     );
-    const stcString = msg["stc_string"];
-    const overlay = A.graphicOverlay(overlayOptions);
+    if (!graphic_options["color"]) graphic_options["color"] = "red";
```

Comment:
why is there a default color here? Does Aladin need it?

Reply:
I checked the code, the overlay takes a color if none is provided so normally no need to define one.

Reply:
I don't know why, but if I give graphic_options as a parameter to A.graphicOverlay without color, the shapes does not appear.
The strange behavior is that none of current nor future shapes will appear. With the following code, there is no shapes on Aladin Lite :
```python
aladin.add_overlay(circle)
aladin.add_overlay(regions, color="orange")
aladin.add_overlay([polygon, rectangle], color="yellow")
```

Reply:
@bmatthieu3 I tried this in the console of the aladin lite demo:

```js
var overlay = A.graphicOverlay({lineWidth: 3});
aladin.addOverlay(overlay);
overlay.add(A.circle(83.66067, 0, 30, {})); 
```

And I got 

```
Uncaught ReferenceError: Color is not defined
    B https://aladin.cds.unistra.fr/AladinLite/api/v3/3.2.0/aladin.js:22
    graphicOverlay https://aladin.cds.unistra.fr/AladinLite/api/v3/3.2.0/aladin.js:61
    <anonymous> debugger eval code:1
```

Reply:
@ManonMarchand - Ah yes, that's an import bug! Should work in a next release.